### ISSUE TITLE
SIlent failure for v0.11

### ISSUE THREAD

**Issue by zepalmer** [314eter/atom-ocaml-merlin#31]
I'm using OCaml 4.02.3 for a project.  Recently, I upgraded to v0.11.2 of this package and discovered that, upon launching Atom, no errors were displayed at all.  On launch, the debug console produced a PID for ocamlmerlin and that process was indeed running, but no amount of keyboard mashing detected any errors.

Installing v0.10.3 of the package results in getting errors immediately, but v0.11.0 does not.  I don't see anything especially suspicious in the [diff](https://github.com/314eter/atom-ocaml-merlin/compare/v0.10.3...v0.11.0) except that the `"providedServices"` key claims a [different version](https://github.com/314eter/atom-ocaml-merlin/compare/v0.10.3...v0.11.0#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L30).  Of course, I'm unfamiliar with the space and so don't have any idea if this is the culprit.

Here's the information that seems relevant:

* OCaml 4.02.3
* ```
  Atom    : 1.7.3
  Electron: 0.36.8
  Chrome  : 47.0.2526.110
  Node    : 5.1.1
  ```
* ```
  Community Packages (11) /home/zpalmer/.atom/packages
  ├── atom-pair@2.0.10
  ├── language-cpp14@0.6.2
  ├── language-latex@1.0.0
  ├── language-ocaml@1.1.2
  ├── language-x86-64-assembly@2.2.8
  ├── linter@1.11.18
  ├── ocaml-indent@0.5.4
  ├── ocaml-merlin@0.10.3
  ├── python-indent@1.0.2
  ├── tab-control@0.6.9
  └── theme-switch@0.4.0
  ```

Please let me know if I can provide additional info.  Thank you for making my coding life easier!

**Comment by 314eter** [comment]
Since v0.11.0, [linter](https://atom.io/packages/linter) v2 is required. That's why the `providedServices` version is different.

**Comment by zepalmer** [comment]
Is there some mechanism for enforcing this in Atom?  It's currently possible to install v0.11.2 without installing linter v2, but I seem to remember something about Atom being pretty lax in enforcing version dependencies.

The source of my problem is apparently that I'm running an older version of Atom (1.7.3); the lack of an automatic update process has hurt me here.  :-/  The newest version of linter requires v1.14.0 or newer, so `apm` was ignoring updates for the linter package.  I think that's what got me into this bind.

Thanks for your help and patience!

**Comment by 314eter** [comment]
I published v0.12.1 for older versions of atom (<1.14.0), and made v0.13.0 require 1.14.0.

**Comment by zepalmer** [comment]
Oo; thank you!
